### ISSUE TITLE
Updated data from get_saq_observations()?

### ISSUE THREAD

**Issue by SverreSolberg** [skgrange/saqgetr#4]
Hi Stuart,
first of all - thanks for an excellent package for extracting EEA data! Very efficient and useful. According to the manual there is a lag of about 1 month for the newest data and indeed the data_processes() and filtered_processes() show this, but when extracting data for the sites with get_saq_observations() I actually get the "almost updated" data (a lag of 3 days from now). Is there a kind of mismatch between these routines or are the data being updated just now, so that I happened to hit the period of monthly updating? According to data_processes() all data end at 2021-01-15 or 2021-01-16, but when extracting data by get_saq_observations() they end at 2021-02-15. (Today is 2021-02-18). 
Thanks.

**Comment by skgrange** [comment]
Hello,
I am glad the package is proving to be useful. 

You are correct, you happen to be querying the data files a few days after I have updated them. The way I have access to the web server means I cannot automate the database updates and file copying at this stage. This means I manually set the tasks running around the middle of the month. Therefore, the data can delayed by a month, but this is not always the case.

Thank you for making me aware of the disconnect between the helper tables and the observations. This is indeed an error. I have been having a lot of trouble with a recently introduced two-factor authentication system. What has probably occurred is the new helper tables were not copied a few days ago and I did not notice. I can confirm these files have been copied again today and will be accessible via the API from tomorrow (~08:00 UTC). I hope that all makes sense,
Stuart.
